doc/modules/ROOT/pages/index.adoc

```diff
@@ -114,12 +114,21 @@ Java::
 NodeJS::
 +
 --
-[source, bash]
+[source, javascript]
 ----
 const { Client } = require('hazelcast-client');
 ----
 --
 
+Python::
++
+--
+[source, python]
+----
+import hazelcast
```

Comment:
Maybe instead of this, we could write here an actual way of installing the client like `pip install hazelcast-python-client`. Because if the user has no prior experience with the client,  they will have to go to our client's documentation to learn the actual package name(`hazelcast-python-client`). This will also be more similar to the Java sample. 

`pip` is not the only way to manage/install dependencies (there are tools like `poetry`, `pipenv` etc.) but the command above should be enough for almost everyone. The users of the other tools could easily install the client once they know the package name.

Reply:
Good idea, updated. I changed it also for NodeJS to be consistent.